src/Avalonia.Visuals/Animation/Animators/BaseBrushAnimator.cs

```diff
@@ -12,10 +15,8 @@ namespace Avalonia.Animation.Animators
     /// redirect them to the properly registered
     /// animators in this class.
     /// </summary>
-    public class BaseBrushAnimator : Animator<IBrush>
+    public class BaseBrushAnimator : Animator<IBrush?>
```

Comment:
Just curious why we changed it to a nullable type?

Reply:
Just because user can actually put null there and we can't do anything with that. 
And there technically can be null, if user used DynamicResource with non-existing resource - it will pass null value to the animation.

Reply:
ahh makes sense 👍